bot/exts/moderation/voice_gate.py

```diff
@@ -88,7 +90,8 @@ async def voice_verify(self, ctx: Context, *_) -> None:
         checks = {
             "verified_at": data["verified_at"] > datetime.utcnow() - timedelta(days=GateConf.minimum_days_verified),
             "total_messages": data["total_messages"] < GateConf.minimum_messages,
-            "voice_banned": data["voice_banned"]
+            "voice_banned": data["voice_banned"],
+            "activity_blocks": data["activity_blocks"] < GateConf.activity_blocks
```

Comment:
The attr is named `minimum_activity_blocks` not `activity_blocks`, sorry I missed this the first time.